nextcord/ui/view.py

```diff
@@ -239,8 +239,17 @@ def from_message(cls, message: Message, /, *, timeout: Optional[float] = 180.0)
 
     @property
     def _expires_at(self) -> Optional[float]:
-        if self.timeout:
-            return time.monotonic() + self.timeout
+        """The monotonic time this view times out at.
+
+        Returns
+        -------
+        Optional[float]
+            When this view times out.
+
+            None if no timeout is set.
+        """
+        if self.timeout and self.__timeout_expiry is not None:
```

Comment:
Why are we checking for None on only one of the checks?
Is the `if self.timeout` check even necessary

Reply:
Looks to me like this can simply return self.__timeout_expiry. If self.timeout is None, then self.__timeout_expiry will be None, and whether self.__timeout_expiry is None or not, returning it will give the correct output.